src/components/Users/index.jsx

```diff
@@ -0,0 +1,58 @@
+import "./User.css";
+import { useContext } from "react";
+import { userContext } from "../../App";
+
+const Users = () => {
+  const { users } = useContext(userContext);
+  return (
+    <>
+      <footer id="site-footer">
+        <div className="container clearfix">
+          <div className="left">
+            <h2 className="subtotal">Users</h2>
+            <input className="search" placeholder="search user" />
+            <h3 className="tax">List of all user who are friend your</h3>
+          </div>
+          <div className="right">
+            <h1 className="total">
+              Total: <span>{users.length}</span>
+            </h1>
+            <p className="btn">Create User</p>
+          </div>
+        </div>
+      </footer>
+      <div>
+        {users?.map((user, index) => {
+          return (
```

Comment:
Do we need to return here?

Reply:
> Do we need to return here?

Yes, actually.

Reply:
Yes we need this return here. In case when we use curly braces with map function then we need return keyword.